friendshipper/src-tauri/src/repo/operations/gh/auto_merge_submit.rs

```diff
@@ -403,6 +417,9 @@ where
                 .delete_branch(&worktree_branch, git::BranchType::Local)
                 .await;
 
+            // unlock all files submitted
+            lock_op.execute().await?;
```

Comment:
bug - the auto merge flow wasn't unlocking files during a stacked commit

Reply:
I thought CI is supposed to handle that?